### PR TITLE
Responsive frontend - GCI

### PR DESCRIPTION
**Pull request by trexdinosaur**
I've placed some notes here on this PR:
https://onenote.com/webapp/pages?token=wp3WppC24IR2CvpDCMcLHWluz6qqmNFtZqkUiNh3dL5n6ck8DpS8lJYYYJ-EcVgEmIJUsjfz5uLWr6inAbU1RVCZfFxK3lNE0&id=636478300879089385

I can copy them in here if needed.

### CODE REVIEW

**Comment by trexdinosaur** [comment]
Note: The update dockerfile commit was mistakenly pushed to the wrong branch. I have since removed it, and it is sitting in #1395.

Thanks.

File: README.md [hunk 1]
```diff
@@ -97,6 +97,13 @@ Follow this guide to setup your development machine.
```
Comment (RishabhJain2018) [inline README.md]: This file should not be modified in this PR as it is not required for the issue. Please remove it.

File: frontend/src/views/web/landing.html [hunk 2]
```diff
@@ -65,9 +65,9 @@ <h4 class="grad-span">
                             <p>
                                 <span class="text-dark-black">{{main.featuredChallenge.short_description}}<span ng-if="main.isMore">...</span></span>
                             </p>
-                            <p>
-                                <span class="w-500 text-light-black">Status: </span> <span class="w-500"> In Progress</span>
-                                <span class="right">
+                            <p style="height: 15px;">
```
Comment (RishabhJain2018) [inline frontend/src/views/web/landing.html]: Please remove `inline css`. All the css must be written in some files under the `css` folder. Also, give the `<p>` tag a class and then write it's css.

**Comment by RishabhJain2018** [comment]
@trexdinosaur Also please `squash` your commits into one single commit. Do let me know if you face any problem in it.

**Comment by trexdinosaur** [comment]
Commit is squashed!

Thanks,
Pavan

**Comment by deshraj** [comment]
@trexdinosaur Can you please post the notes here along with the original screenshot of the modified interface?

**Comment by trexdinosaur** [comment]
image

image
image
image
image
Only change that is not pictured is the status moved to the left.

**Comment by aka-jain** [comment]
@trexdinosaur, hey is your GCI task approved?

**Comment by trexdinosaur** [comment]
Yep, it was approved.

________________________________

**Comment by aka-jain** [comment]
Closing this PR as these changes has already done.

**Pull request closed without merging**